SPECS/libnvidia-container/libnvidia-container.spec

```diff
@@ -0,0 +1,273 @@
+%define modprobe_version 450.57
+%define _major 1
+Summary:        NVIDIA container runtime library
+Name:           libnvidia-container
+Version:        1.3.3
+Release:        2%{?dist}
+License:        BSD-3-Clause AND Apache-2.0 AND GPL-3.0-or-later AND LGPL-3.0-or-later AND MIT AND GPL-2.0-only
```

Comment:
Wherever possible, please use [Fedora's short names](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses) for the license.